src/java/com/phenix/pct/CompilationAttributes.java

```diff
@@ -415,6 +441,10 @@ protected void writeCompilationProcedure(File f, Charset c) {
             bw.newLine();
             bw.write("DEFINE INPUT PARAMETER ipListing AS CHARACTER NO-UNDO.");
             bw.newLine();
+            bw.write("DEFINE INPUT PARAMETER ipPageSize AS INTEGER NO-UNDO.");
```

Comment:
No need to pass the parameters here, as the procedure is generated on the fly, so you insert the values at line 485 only if parameters are set.